src/main/java/br/com/dandrade/viagens/controllers/CountryController.java

```diff
@@ -0,0 +1,37 @@
+package br.com.dandrade.viagens.controllers;
+
+import br.com.dandrade.viagens.models.Country;
+import br.com.dandrade.viagens.repository.CountryRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping(value = "/api/country",
+        produces = MediaType.APPLICATION_JSON_VALUE,
+        consumes = MediaType.APPLICATION_JSON_VALUE)
+public class CountryController {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CountryRepository.class);
+
+    private CountryRepository repository;
+
+
+    public CountryController(CountryRepository repository) {
+        this.repository = repository;
+    }
+
+    @PostMapping()
+    @ResponseStatus(code = HttpStatus.OK)
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*
Legal o uso do ResponseStatus. Por outro lado, se você não define explicitamente, o default já é 200 :).

Reply:
Nesse caso deixei o ResponseStatus porque na primeira vez eu coloquei com o status 201, mas quando olhei a tarefa dizia 200 ai fiz o mais difícil rsrs